premake5.lua

```diff
@@ -278,6 +278,7 @@ if premake.action.supports("None") then
 		kind "None"
 
 		files "website/**"
+		removefiles "website/node_modules/**" -- 400+ generated files
```

Comment:
Not sure if this comment is necessary. I would say that "400+" is not very accurate as well. My `node_modules` folder has 32759 files. Yes, it is more than 400, but you know what I mean 😄

Reply:
I understand more the slowdown then ;-)